sdk/kusto/arm-kusto/src/operations/dataConnections.ts

```diff
@@ -72,35 +72,9 @@ export class DataConnections {
    * @param [options] The optional parameters
    * @returns Promise<Models.DataConnectionsDataConnectionValidationMethodResponse>
    */
-  dataConnectionValidationMethod(resourceGroupName: string, clusterName: string, databaseName: string, parameters: Models.DataConnectionValidation, options?: msRest.RequestOptionsBase): Promise<Models.DataConnectionsDataConnectionValidationMethodResponse>;
-  /**
-   * @param resourceGroupName The name of the resource group containing the Kusto cluster.
-   * @param clusterName The name of the Kusto cluster.
-   * @param databaseName The name of the database in the Kusto cluster.
-   * @param parameters The data connection parameters supplied to the CreateOrUpdate operation.
-   * @param callback The callback
-   */
-  dataConnectionValidationMethod(resourceGroupName: string, clusterName: string, databaseName: string, parameters: Models.DataConnectionValidation, callback: msRest.ServiceCallback<Models.DataConnectionValidationListResult>): void;
-  /**
-   * @param resourceGroupName The name of the resource group containing the Kusto cluster.
-   * @param clusterName The name of the Kusto cluster.
-   * @param databaseName The name of the database in the Kusto cluster.
-   * @param parameters The data connection parameters supplied to the CreateOrUpdate operation.
-   * @param options The optional parameters
-   * @param callback The callback
-   */
-  dataConnectionValidationMethod(resourceGroupName: string, clusterName: string, databaseName: string, parameters: Models.DataConnectionValidation, options: msRest.RequestOptionsBase, callback: msRest.ServiceCallback<Models.DataConnectionValidationListResult>): void;
-  dataConnectionValidationMethod(resourceGroupName: string, clusterName: string, databaseName: string, parameters: Models.DataConnectionValidation, options?: msRest.RequestOptionsBase | msRest.ServiceCallback<Models.DataConnectionValidationListResult>, callback?: msRest.ServiceCallback<Models.DataConnectionValidationListResult>): Promise<Models.DataConnectionsDataConnectionValidationMethodResponse> {
-    return this.client.sendOperationRequest(
-      {
-        resourceGroupName,
-        clusterName,
-        databaseName,
-        parameters,
-        options
-      },
-      dataConnectionValidationMethodOperationSpec,
-      callback) as Promise<Models.DataConnectionsDataConnectionValidationMethodResponse>;
+  dataConnectionValidationMethod(resourceGroupName: string, clusterName: string, databaseName: string, parameters: Models.DataConnectionValidation, options?: msRest.RequestOptionsBase): Promise<Models.DataConnectionsDataConnectionValidationMethodResponse> {
```

Comment:
Is this a breaking change compared with https://github.com/Azure/azure-sdk-for-js/pull/10043/files#diff-4827172be7bd11a60faaafc3171e12c4L92